runtime/frequency/src/xcm/tests/check_whitelist.rs

```diff
@@ -0,0 +1,23 @@
+use crate::*;
+use frame_support::traits::WhitelistedStorageKeys;
+use sp_core::hexdisplay::HexDisplay;
+use std::collections::HashSet;
+
+#[test]
+fn check_whitelist() {
+	let whitelist: HashSet<String> = dbg!(AllPalletsWithSystem::whitelisted_storage_keys()
+		.iter()
+		.map(|e| HexDisplay::from(&e.key).to_string())
+		.collect());
+
+	// Block Number
+	assert!(whitelist.contains("26aa394eea5630e07c48ae0c9558cef702a5c1b19ab7a04f536c519aca4983ac"));
```

Comment:
It would be helpful to include a convenient link or path to verify these values.